composer.json

```diff
@@ -29,6 +29,7 @@
     }
     , "require-dev": {
         "phpunit/phpunit": "4.1.*"
+        , "cakephp/cakephp-codesniffer": "2.*"
```

Comment:
I will - was gonna make my life easier and do so anyways. Just because it's JSON produced by PHP but generally speaking, [comma should be first](https://docs.npmjs.com/misc/coding-style#comma-first).